### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nmarquesin**
docs: add a session to "Submitting a pull request" describing what the contributor can expect from the code maintainers after submitting a pull request.

## Description:
TotalCross code maintainers commit to reply to open pull requests no later than 3 days and to discuss it during the next weekly backlog grooming meeting.

## Motivation and Context:
Added paragraphs provide a guideline for code maintainers to reply to pull requests, both for first interaction and when backlog discussions will take place.
It also aligns code contributors expectations for come maintainers replies.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -196,4 +195,10 @@ Before submitting a pull request, please make sure the following is done:
...
 - Build your changes;
 - If you’ve fixed a bug or added code that should be tested, **add tests**;
-- Ensure your commit is validated;
\ No newline at end of file
+- Ensure your commit is validated;
+
+### What happens next?
+
+After your pull request has been submitted, TotalCross code maintainers will acknowledge it as soon as possible. We aim to do it in three days tops.
+
```
Comment (acmlira) [inline CONTRIBUTING.md:203]: We can add a table to make it more graphical and _flashy_?

Comment (nmarquesin) [reply CONTRIBUTING.md:203]: That's a good idea! Something like this?

| Action | Action deadline |
|---|---|
| Pull request committed | -  |
| First reply | 3 days  | 
| Backlog prioritisation feedback | 7 days  |

Comment (acmlira) [reply CONTRIBUTING.md:203]: Yep, I think we are already in a good first version!

**Review by acmlira — DISMISSED** [review]

**Review by ricardobna — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (acmlira) [reply CONTRIBUTING.md:203]: Hey @nmarquesin **[this](https://github.com/TotalCross/totalcross/pull/144#discussion_r490436881)** need to be committed :sweat_smile:

Comment (nmarquesin) [reply CONTRIBUTING.md:203]: Ops! I got confused by the commit message error and forgot to do it properly. Should be fixed now 👍

**Review by acmlira — APPROVED** [review]

**Pull request merged**